src/bundle/Resources/views/content/tab/versions/tab.html.twig

```diff
@@ -106,6 +113,7 @@
 {% block javascripts %}
     {% javascripts
     '@EzPlatformAdminUiBundle/Resources/public/js/scripts/button.state.toggle.js'
+    '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.content.draft.edit.js'
```

Comment:
I'm not sure if `content draft edit` is a proper name. I'm thinking about next feature `edit conflict` and we will have to edit conflicts. This should be something like `version edit conflict`.